core/trino-main/src/main/java/io/trino/sql/analyzer/FeaturesConfig.java

```diff
@@ -135,15 +135,6 @@
     private DataSize filterAndProjectMinOutputPageSize = DataSize.of(500, KILOBYTE);
     private int filterAndProjectMinOutputPageRowCount = 256;
     private int maxGroupingSets = 2048;
-    /*
-     * We leave join pushdown disabled by default as this is safer option.
-     * Pushing down a join which substantially increases the row count vs
-     * sizes of left anf right table separately, may incur huge cost both
-     * in terms of performance and money due to an increased network traffic.
-     *
-     * As soon as we have cost based logic to determine join pushdown feasibility we
-     * will change the default to automatic.
-     */
```

Comment:
Nice.

Reply:
Actually I should have left the initial part of the comment I think.

Reply:
in this PR -- sure. But ultimately we want to enable this by default in engine, as we transfer the responsibility onto connectors.